Only in libtorch-1.4.0/: libtorch
diff -ru libtorch/share/cmake/Caffe2/public/cuda.cmake libtorch-1.4.0/share/cmake/Caffe2/public/cuda.cmake
--- libtorch/share/cmake/Caffe2/public/cuda.cmake	2020-01-14 11:35:56.000000000 -0800
+++ libtorch-1.4.0/share/cmake/Caffe2/public/cuda.cmake	2020-09-22 11:01:00.791427176 -0700
@@ -147,7 +147,11 @@
 # ---[ Extract versions
 if(CAFFE2_USE_CUDNN)
   # Get cuDNN version
-  file(READ ${CUDNN_INCLUDE_PATH}/cudnn.h CUDNN_HEADER_CONTENTS)
+  if(EXISTS ${CUDNN_INCLUDE_PATH}/cudnn_version.h) 
+      file(READ ${CUDNN_INCLUDE_PATH}/cudnn_version.h CUDNN_HEADER_CONTENTS) 
+  else() 
+      file(READ ${CUDNN_INCLUDE_PATH}/cudnn.h CUDNN_HEADER_CONTENTS) 
+  endif() 
   string(REGEX MATCH "define CUDNN_MAJOR * +([0-9]+)"
                CUDNN_VERSION_MAJOR "${CUDNN_HEADER_CONTENTS}")
   string(REGEX REPLACE "define CUDNN_MAJOR * +([0-9]+)" "\\1"
